doc/secondary-sources.rst

```diff
@@ -36,6 +36,31 @@ Linear array
 
     Linear loudspeaker array with a length of 3m consiting of 21 loudspeakers.
 
+You can also logarithmically space the secondary sources for a linear array
```

Comment:
maybe: You can also create logarithmic spacing for a linear array

Reply:
Thanks, I changed this.